client/checkout/woopay/express-button/express-checkout-iframe.js

```diff
@@ -209,33 +207,32 @@ export const expressCheckoutIframe = async ( api, context, emailSelector ) => {
 	} );
 
 	window.addEventListener( 'message', ( e ) => {
-		if ( ! getConfig( 'platformCheckoutHost' ).startsWith( e.origin ) ) {
+		if ( ! getConfig( 'woopayHost' ).startsWith( e.origin ) ) {
 			return;
 		}
 
 		switch ( e.data.action ) {
 			case 'otp_email_submitted':
 				userEmail = e.data.userEmail;
 				break;
-			case 'redirect_to_platform_checkout':
+			case 'redirect_to_woopay':
```

Comment:
These action names are coming from WooPay server so would require coordination there to work. For now what about keeping both cases for now? I'm unsure how we'll ultimately handle the backwards compatibility here though longer term, so open to other suggestions.